common/stacks/src/main/scala/uk/ac/wellcome/platform/api/common/services/source/CatalogueSource.scala

```diff
@@ -5,14 +5,12 @@ import akka.http.scaladsl.model.Uri
 import akka.http.scaladsl.model.Uri.Path
 import uk.ac.wellcome.platform.api.common.models.Identifier
 import uk.ac.wellcome.platform.api.http.AkkaClientGet
-import weco.catalogue.internal_model.identifiers.CanonicalId
 
```

Comment:
What's remaining before we can get rid of this entirely?

Reply:
Looking up the items in the holds you already have, which I'll do in the final PR of this batch.